### ISSUE TITLE
Make number of Python worker processes configurable

### ISSUE THREAD

**Issue by kkovaacs** [eqlabs/pathfinder#354]
RIght now we have the number of Python subprocesses hard-coded (at 2), and for use cases where a lot of `starknet_call` JSON-RPC calls are made this might actually be a limitation.

We should make the number of subprocesses configurable.

**Comment by xJonathanLEI** [comment]
Does a `starknet_call` request lock a subprocess though?

**Comment by Mirko-von-Leipzig** [comment]
> Does a `starknet_call` request lock a subprocess though?

Yes; we effectively have a maximum of 2 `starknet_call`s which can execute concurrently. Any further calls will be queued and only processed once those are done.

Spinning python processes up lazily is quite expensive, as just launching costs several seconds on a good machine. In addition, allowing a launch per `starknet_call` would be an attack vector, so this must be limited somehow.

A limit harcdcoded to 2 is a bit little though.

**Comment by Mirko-von-Leipzig** [comment]
Big reason why we really want a Rust VM. In addition to the python packaging issues.

**Comment by xJonathanLEI** [comment]
Wow that's quite bad...
